app/views/application/_header.html.erb

```diff
@@ -28,3 +30,18 @@
     <% end %>
   </div>
 </header>
+
+<% if respond_to?(:nav_items) %>
```

Comment:
I don't quite like that we use `respond_to` for this. Could we define a default that's nil or empty, and then use `if nav_items.present?` instead?

Reply:
Yeah I can define an empty array as we will probably want to add nav items to the main app at some point anyway